static/js/components/GroupSources.jsx

```diff
@@ -0,0 +1,349 @@
+import React, { useEffect, Suspense } from "react";
+import PropTypes from "prop-types";
+import { useSelector, useDispatch } from "react-redux";
+
+import TableCell from "@material-ui/core/TableCell";
+import TableRow from "@material-ui/core/TableRow";
+
+import Typography from "@material-ui/core/Typography";
+import IconButton from "@material-ui/core/IconButton";
+import Grid from "@material-ui/core/Grid";
+import Chip from "@material-ui/core/Chip";
+import Link from "@material-ui/core/Link";
+import PictureAsPdfIcon from "@material-ui/icons/PictureAsPdf";
+
+import MUIDataTable from "mui-datatables";
+import { makeStyles } from "@material-ui/core/styles";
+
+import Tooltip from "@material-ui/core/Tooltip";
+import GroupIcon from "@material-ui/icons/Group";
+
+import dayjs from "dayjs";
+
+import { ra_to_hours, dec_to_hours } from "../units";
+import * as SourcesAction from "../ducks/sources";
+import styles from "./GroupSources.css";
+import ThumbnailList from "./ThumbnailList";
+import UserAvatar from "./UserAvatar";
+import ShowClassification from "./ShowClassification";
+
+const VegaPlot = React.lazy(() => import("./VegaPlot"));
+
+const useStyles = makeStyles((theme) => ({
```

Comment:
incorporate CSS into this

Reply:
Is there any other page where the comments CSS have been incorporated into the makeStyles? I could really use a working example.